src/plugins/encryption/Encryption.h

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
What's this class for? Is it still needed as a placeholder or can it be discarded?